### PR TITLE
RHDEVDOCS-6276: Update OCP docs to reflect that Shared Resource CSI Driver is removed from OCP but is GA in Builds

### PR DESCRIPTION
**Pull request by shivanisathe25**
Version(s):

Issue: https://issues.redhat.com/browse/RHDEVDOCS-6276

Link to docs preview:

QE review: @adambkaplan , @siamaksade 
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by avinal — APPROVED** [review]
Looks Okay To Me.

**Comment by Preeticp** [comment]
CS Ack

File: release_notes/ocp-4-17-release-notes.adoc [hunk 1]
```diff
@@ -1474,8 +1474,16 @@ In the following tables, features are marked with the following statuses:
 |General Availability
 |Removed
 |Removed
-|====
 
+|Shared Resources CSI Driver in OpenShift Builds
+|Deprecated
+|Deprecated
+|Deprecated ^[1]^
```
Comment (avinal) [inline release_notes/ocp-4-17-release-notes.adoc:1481]: This should not be deprecated In OpenShift Builds, right?, It should be deprecated in OCP

**Review by avinal — CHANGES_REQUESTED** [review]
New changes have some issues. Please see the comments.

**Review by avinal — APPROVED** [review]
Seems Good to Me

**Review by adambkaplan — CHANGES_REQUESTED** [review]
Shouldn't this PR target the docs for OCP 4.18? The corresponding features in OCP 4.17 are _deprecated_ - it is in 4.18 where the Shared Resource CSI driver is _removed_.

**Comment by reestr** [comment]
LGTM

**Review by adambkaplan — APPROVED** [review]
LGTM

Took me a bit to piece together how the changes would impact the docs - I probably should have walked through the preview pages first!

All look good to me. :+1:

**Comment by ILpinto** [comment]
LGTM

**Comment by kavverma** [comment]
LGTM

File: release_notes/ocp-4-17-release-notes.adoc [hunk 2]
```diff
@@ -1474,8 +1474,16 @@ In the following tables, features are marked with the following statuses:
...
 
+|Shared Resources CSI Driver ^[1]^
+|Deprecated
+|Deprecated
+|Deprecated 
+|====
+[.small]
+--
+1. The Shared Resource CSI Driver feature is now generally available in Builds for Red Hat OpenShift 1.1. This feature is now deprecated in OpenShift Container Platform. To use this feature, ensure you are using Builds for Red Hat OpenShift 1.1 or a more recent version.
+--
```
Comment (maxwelldb) [inline release_notes/ocp-4-17-release-notes.adoc:1486]: Is there a way to call out this information without this footnote + style?

File: release_notes/ocp-4-17-release-notes.adoc [hunk 3]
```diff
@@ -2295,6 +2298,7 @@ In the following tables, features are marked with the following statuses:
 |Not Available
 |Developer Preview
 
+
```
Comment (maxwelldb) [inline release_notes/ocp-4-17-release-notes.adoc:2301]:
```suggestion
```

**Review by maxwelldb — APPROVED** [review]
Looks fine to me. A comment and a suggestion.

**Comment by lpettyjo** [comment]
> Shouldn't this PR target the docs for OCP 4.18? The corresponding features in OCP 4.17 are _deprecated_ - it is in 4.18 where the Shared Resource CSI driver is _removed_.

I'm the storage writer for OCP. I have a JIRA card to make these changes in 4.18. https://issues.redhat.com/browse/OSDOCS-12842

File: release_notes/ocp-4-17-release-notes.adoc (hunk 2 above)
Comment (shivanisathe25) [reply release_notes/ocp-4-17-release-notes.adoc:1486]: Hi @maxwelldb , please see table 9 in https://docs.openshift.com/container-platform/4.15/release_notes/ocp-4-15-release-notes.html#ocp-4-15-deprecated-removed-features

We decided to follow the same method.

**Pull request merged**